libs/partners/google-genai/langchain_google_genai/chat_models.py

```diff
@@ -619,3 +619,18 @@ async def _astream(
             yield gen
             if run_manager:
                 await run_manager.on_llm_new_token(gen.text)
+
+    def _prepare_chat(
+        self,
+        messages: List[BaseMessage],
+        stop: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> Tuple[Dict[str, Any], genai.ChatSession, genai.types.ContentDict]:
+        params = self._prepare_params(stop, **kwargs)
+        history = _parse_chat_history(
+            messages,
+            convert_system_message_to_human=self.convert_system_message_to_human,
+        )
+        message = history.pop()
+        chat = self.client.start_chat(history=history)
```

Comment:
This will throw an error if `candidate_count` > 1, right? Is there any real benefit to using this, given that the chat history is managed externally anyway?

Reply:
`candidate_count > 1` is unsupported at this moment anyways, `generate_content` would throw the same error